src/Aspire.Dashboard/Components/Controls/GridValue.razor.css

```diff
@@ -20,6 +20,6 @@
     cursor: pointer;
 }
 
-::deep:hover .defaultHidden {
+::deep:hover .defaultHidden, ::deep:focus-within .defaultHidden  {
```

Comment:
nit: I prefer to keep selectors on separate lines for easier scanning

Reply:
personally it confuses me more but I’m happy to change